hypothesis-python/src/hypothesis/internal/conjecture/junkdrawer.py

```diff
@@ -216,3 +216,41 @@ def swap(ls, i, j):
     if i == j:
         return
     ls[i], ls[j] = ls[j], ls[i]
+
+
+def find_integer(f):
```

Comment:
This is the new home for `find_integer`, but I think you forgot to delete the old one in `common.py`?